pyswarms/utils/plotters/formatters.py

```diff
@@ -57,10 +58,15 @@ class Designer(object):
     )
     legend = attrib(validator=instance_of(str), default="Cost")
     label = attrib(
-        validator=instance_of((str, list, tuple)), default=["x-axis", "y-axis"]
+        validator=instance_of((str, list, tuple)),
+        default=["x-axis", "y-axis", "z-axis"],
     )
     limits = attrib(
-        validator=instance_of((list, tuple)), default=[(-1, 1), (-1, 1)]
+        validator=instance_of((list, tuple)), default=[(-1, 1), (-1, 1), (-1, 1)]
+    )
+    colormap = attrib(
+        validator=instance_of((type(cm.viridis), type(cm.binary))),
+        default=cm.viridis,
     )
```

Comment:
This line of code is related to my comment above. I think it's better to check against `matplotlib.colors.Colormap` because it is the base class for whatever `type(cm.viridis)` and `type(cm.binary)` is